pkg/assetfetcher/http.go

```diff
@@ -82,6 +82,12 @@ func (c *NexposeAssetFetcher) FetchAssets(ctx context.Context, siteID string) (<
 		return nil, errChan
 	}
 
+	if res.StatusCode != http.StatusOK {
```

Comment:
question: Did you mean for this content to be included? It does not appear related to the stats problem

Reply:
True, I figured it was small enough to add into this PR. I'll separate it out into it's own PR.

Reply:
Fixed.